.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## Bug Report
+<!--
+    If you are not submitting a bug report, ignore this template and see
+      http://Chapel-Issues-Best-Practices.com  (TODO)
```

Comment:
What if we were to change this to "If you are not submitting a bug report, feel free to erase this template and describe your issue using markdown.  For more information on submitting bugs and issues refer to ..."

And yeah, I agree that this ought to point to bugs.html (primarily because our compiler does, so I don't see that page going away overnight).  Note that it should probably be chapel.cray.com/bugs.html for brevity, ease of remembering it, and to support our ability to move it easily).